layouts/css/page-modules/_home.scss

```diff
@@ -5,18 +5,21 @@
 }
 
 .home-version-banner {
-  color: $node-gray;
+  // color: $node-gray;
   font-size: 130%;
-  background-color: $light-green;
+  //background-color: $light-green;
```

Comment:
Might be better to just remove instead of commenting out in one commit, then it can be rolled back later if desired

Reply:
I had to `roll back` for this PR and it was honestly a PITA. We totally can just remove it but I figured it was faster, even if messier, to just leave the old style there.

TBH a better approach would be to make an entirely different banner... 🤔

Reply:
This isn't a blocker. Since it's sass, you could probably extend with a new CSS selector, but I haven't really looked to see how the current one is setup to see if that makes any sense

Reply:
fixed PTAL